components/brave_new_tab_ui/storage/new_tab_storage.ts

```diff
@@ -40,6 +41,7 @@ export const defaultState: NewTab.State = {
   showEmptyPage: false,
   braveRewardsSupported: false,
   braveTalkSupported: false,
+  braveVPNSupported: loadTimeData.getBoolean('vpnWidgetSupported'),
```

Comment:
do we still need this?

Reply:
Deleted from state and added it as a `NewTabPage`'s member var as it's the only place that use it.